rero_ils/filter.py

```diff
@@ -184,3 +189,19 @@ def message_filter(key):
     :return: none or a json (check structure into the class Message).
     """
     return Message.get(key)
+
+
+def translate(data, prefix='', separator=', '):
```

Comment:
I don't know if it's a good idea to override the basic Jinja `translate` pipe (working with babelex). This is why we use `translat` function name before... Need a deep check during US tests